bedrock/pocket/templates/pocket/updates-signup.html

```diff
@@ -32,12 +32,63 @@
         }
       ),
     ) %}
-        <h1>Keep up with Pocket</h1>
-        <p>To learn about future Pocket updates and how you can be involved, please reach out to <a href="mailto:press@pocket.com">press@pocket.com</a>.</p>
-    {% endcall %}
+      <div id="pocket-updates-thanks" class="pocket-updates-success hidden">
+        <h1>{{ ftl('pocket-updates-signup-success-done-great-reads-on-way') }}</h1>
+        <p>{{ ftl('pocket-updates-signup-success-follow-on') }}</p>
+        <a class="mzp-c-button" href="/{{LANG}}/explore">
+          {{ ftl('pocket-updates-signup-success-discover-must-read-articles') }}
+        </a>
+      </div>
+
+
+      <div id="pocket-updates-form-wrapper">
+        <h1>{{ ftl('pocket-updates-signup-keep-up-with-pocket') }}</h1>
+        <p>{{ ftl('pocket-updates-signup-join-our-mailing-list') }}</p>
+        <form action="{{ url('pocket.newsletter-subscribe') }}" class="mzp-c-newsletter-form pocket-updates-form" id="updates-form">
+
+          <div class="mzp-c-form-errors hide-from-legacy-ie hidden" id="subscribe-errors">
+            <ul class="mzp-u-list-styled">
+              <li class="error-email-invalid hidden">
+                {{ ftl('pocket-updates-form-please-enter-a-valid') }}
+              </li>
+              <li class="error-try-again-later hidden">
+                {{ ftl('pocket-updates-form-we-are-sorry-but-there') }}
+              </li>
+            </ul>
+          </div>
+
+          <fieldset>
+            <label for="email" class="visually-hidden">
+              {{ ftl('pocket-udpates-your-email-address') }}
+            </label>
+            <input type="email" name="email" id="email" placeholder="{{ ftl('pocket-updates-your-email-address') }}" required aria-required="true">
```

Comment:
I think we're ok just to use `required` today
```suggestion
            <input type="email" name="email" id="email" placeholder="{{ ftl('pocket-updates-your-email-address') }}" required>
```